db/seeds/development/base.seeds.rb

```diff
@@ -16,47 +16,133 @@
               ])
-SprintPosition.create!([
-                           {position: 2, sprint_id: 1, story_id: 1},
-                           {position: 1, sprint_id: 1, story_id: 2},
-                           {position: 3, sprint_id: 2, story_id: 9},
-                           {position: 4, sprint_id: 2, story_id: 12},
-                           {position: 2, sprint_id: 2, story_id: 12},
-                           {position: 1, sprint_id: 2, story_id: 11}
-                       ])
+
+# Project 2
+Story.create!([
+                  {title: "Als​ ​ Student,​ ​ möchte​ ​ ich​ ​ über​ ​ eine Modulübersicht​ ​ genauere Informationen​ ​ über​ ​ eine Lehrveranstaltung​ ​ bei​ ​ einem​ ​ Dozenten angezeigt​ ​ bekommen.",
+                   description: nil, status: 0, points: nil, sprint_id: nil, project_id: 2, topic_id: 1},
+                  {title: "Als​ ​ Student,​ ​ möchte​ ​ ich​ ​ Module​ ​ nach Studiengang​ ​ / ​ ​ Semester​ ​ gefiltert angezeigt​ ​ bekommen.",
+                   description: nil, status: 0, points: nil, sprint_id: nil, project_id: 2, topic_id: 2},
+                  {title: "Als​ ​ Nutzer​ ​ möchte​ ​ ich​ ​ nach​ ​ einem bestimmten​ ​ Kurs​ ​ suchen​ ​ können.",
+                   description: nil, status: 0, points: nil, sprint_id: nil, project_id: 2, topic_id: 3},
+                  {title: "Als​ ​ Dozent,​ ​ kann​ ​ ich​ ​ eine Beschreibung​ ​ zu​ ​ meinem​ ​ Kurs hochladen.",
+                   description: nil, status: 0, points: nil, sprint_id: nil, project_id: 2, topic_id: 3},
+                  {title: "Als​ ​ nicht​ ​ autorisierter​ ​ Nutzer,​ ​ möchte ich​ ​ ein​ ​ Account​ ​ erstellen​ ​ können.",
+                   description: nil, status: 0, points: nil, sprint_id: nil, project_id: 2, topic_id: 2},
+                  {title: "Als​ ​ autorisierter​ ​ Nutzer,​ ​ möchte​ ​ ich mich​ ​ anmelden​ ​ können.",
+                   description: nil, status: 0, points: nil, sprint_id: nil, project_id: 2, topic_id: 2},
+                  {title: "Als​ ​ Nutzer,​ ​ möchte​ ​ ich​ ​ die​ ​ Möglichkeit haben​ ​ zu​ ​ sehen,​ ​ wie​ ​ andere​ ​ den​ ​ Kurs in​ ​ einem​ ​ Metriksystem​ ​ charakterisiert haben.",
+                   description: nil, status: 0, points: nil, sprint_id: nil, project_id: 2, topic_id: 4},
+                  {title: "Als​ ​ autorisierter​ ​ Nutzer​ ​ (Student),​ ​ kann ich​ ​ meine​ ​ Erfahrungen​ ​ mit​ ​ dem​ ​ Kurs mit​ ​ anderen​ ​ teilen,​ ​ indem​ ​ ich Bewertungen​ ​ für​ ​ diesen​ ​ Kurs​ ​ im Metriksystem​ ​ eingebe.",
+                   description: nil, status: 0, points: nil, sprint_id: nil, project_id: 2, topic_id: 4},
+                  {title: "Als​ ​ Nutzer​ ​ möchte​ ​ ich​ ​ alle​ ​ verfügbaren Kurse​ ​ sehen​ ​ können.",
+                   description: nil, status: 0, points: nil, sprint_id: nil, project_id: 2, topic_id: 5},
+              ])
+
 ProjectPosition.create!([
```

Comment:
@anjakammer creating the project and sprint positions explicitly results in them being double in the DB (two entries per story/sprint), which leads to all the sorting problems we have, since the sorting order is pretty much random. I've already fixed everything sorting related in the existing seeds in the story/interface_improvements branch, but please already remove the explicitly created positions here as well